### PR TITLE
TS-540: Changes after wordpress review

### PR DESCRIPTION
**Pull request by KubaJM**
Rename woocommerce sentences to e-commerce as domain text, sanitized input and output variables, refactorHandle local environment url for webhooks and redirects

### CODE REVIEW

File: includes/class-wc-getkevin-gateway.php [hunk 1]
```diff
@@ -569,50 +566,66 @@ public function process_refund($order_id, $amount = null, $reason = '')
...
-            return new WP_Error('error', __('Refund failed.', 'woocommerce-gateway-getkevin'));
+            return new WP_Error('error', __('Refund failed.', 'e-commerce-gateway-getkevin'));
         }
     }
 
     public function order_received_text($text, $order)
     {
         if ($order && $this->id === $order->get_payment_method()) {
-            $statusGroup = $_GET['statusGroup'] ?? null;
+            $statusGroup = !empty($_GET['statusGroup']) ? sanitize_text_field($_GET['statusGroup']) : null;
```
Comment (KubaJM) [inline includes/class-wc-getkevin-gateway.php:586]: we have to keep reverse compatibility with 5.6 PHP so ?? is not accepted

File: getkevin.php [hunk 2]
```diff
@@ -5,13 +5,13 @@
  * Description: kevin. is a payment infrastructure company which offers payment initiation service in EU&EEA.
  * Version: 3.0.0
  * Requires at least: 5.2
- * Tested up to: 5.5
+ * Tested up to: 6.0
```
Comment (saltanovas) [inline getkevin.php:8]: 6.0 has beta compatibility with PHP8. As long as I know, we have some issues with it: https://github.com/getkevin/kevin-woocommerce/issues/12

Did you look into it?

Comment (KubaJM) [reply getkevin.php:8]: I will check it

File: includes/class-wc-getkevin-gateway.php (hunk 1 above)
Comment (KubaJM) [reply includes/class-wc-getkevin-gateway.php:586]: Indeed, this with the switch is a better one. I don't want to refactor here that`s why it was simply sanitize

**Review by tomas-maciulis — APPROVED** [review]

File: getkevin.php (hunk 2 above)
Comment (tomas-maciulis) [reply getkevin.php:9]: image

I'm getting this error after selecting bank and clicking "Place Order" in checkout page. I'm using the latest official wordpress docker image `6.0.0-apache` and latest WooCommerce version 6.6.1

**Pull request merged**